ros-examples/src/aero_offboard_velocity/src/aero_offboard_velocity_node.cpp

```diff
@@ -0,0 +1,355 @@
+    ros::Duration(0.01).sleep();
+  }
+  ROS_INFO("Done");
+
+  ROS_INFO("wait for a bit");
+  for (int i = 200; ros::ok() && i > 0; --i)
+  {
+    set_Offboard_Velocity_Body(0.0f, 0.0f, 0.0f, 0.0f);
+    ros::Duration(0.01).sleep();
+  }
+  ROS_INFO("Done");
+
+  ROS_INFO("fly a circle");
+  for (int i = 500; ros::ok() && i > 0; --i)
+  {
+    set_Offboard_Velocity_Body(5.0f, 5.0f, 0.0f, -60.0f);
+    ros::Duration(0.01).sleep();
+  }
+  ROS_INFO("Done");
+
+  ROS_INFO("wait for a bit");
+  for (int i = 200; ros::ok() && i > 0; --i)
+  {
+    set_Offboard_Velocity_Body(0.0f, 0.0f, 0.0f, 0.0f);
+    ros::Duration(0.01).sleep();
+  }
+  ROS_INFO("Done");
+
+  ROS_INFO("fly a circle sideways");
+  for (int i = 500; ros::ok() && i > 0; --i)
+  {
+    set_Offboard_Velocity_Body(0.0f, 5.0f, 0.0f, -60.0f);
+    ros::Duration(0.01).sleep();
+  }
+  ROS_INFO("Done");
+
+  ROS_INFO("wait for a bit");
+  for (int i = 500; ros::ok() && i > 0; --i)
+  {
+    set_Offboard_Velocity_Body(0.0f, 0.0f, 0.0f, 0.0f);
+    ros::Duration(0.01).sleep();
+  }
+  ROS_INFO("Done");
+}
+int main(int argc, char** argv)
+{
+  ros::init(argc, argv, "offb_velocity_node");
+  ros::NodeHandle nh;
+
+  // the setpoint publishing rate MUST be faster than 2Hz
+  ros::Rate rate(20.0);
+  Offboard offb_obj(&nh);
+
+  boost::thread thread_mode(wait_For_FCU_Connection);
+
+  // arming
+  ros::ServiceClient arming_client = nh.serviceClient<mavros_msgs::CommandBool>("mavros/cmd/arming");
+  mavros_msgs::CommandBool srv_arm;
+  srv_arm.request.value = true;
+  if (arming_client.call(srv_arm) && srv_arm.response.success)
+    ROS_INFO("ARM sent %d", srv_arm.response.success);
+  else
+  {
+    ROS_ERROR("Failed arming/disarming");
+    return -1;
+  }
+
+  // takeoff
+  ros::ServiceClient takeoff_client = nh.serviceClient<mavros_msgs::CommandTOL>("/mavros/cmd/takeoff");
+  mavros_msgs::CommandTOL srv_takeoff;
+  srv_takeoff.request.altitude = 490.0;
+  srv_takeoff.request.latitude = 47.3977415;
+  srv_takeoff.request.longitude = 8.5455937;
+  srv_takeoff.request.min_pitch = 0;
+  srv_takeoff.request.yaw = 0;
+  if (takeoff_client.call(srv_takeoff) && srv_takeoff.response.success)
+    ROS_INFO("takeoff sent %d", srv_takeoff.response.success);
+  else
+  {
+    ROS_ERROR("Failed Takeoff");
+    return -1;
+  }
+
+  sleep(5);
+
+  // offboard ned velocity
+  bool ret = offb_obj.offb_Ctrl_Ned();
+  if (ret == true)
+  {
+    printf("Done with Ned velocity");
+  }
+
+  // land
+  ros::ServiceClient land_client = nh.serviceClient<mavros_msgs::CommandTOL>("/mavros/cmd/land");
+  mavros_msgs::CommandTOL srv_land{};
+  if (land_client.call(srv_land) && srv_land.response.success)
+    ROS_INFO("land sent %d", srv_land.response.success);
+  else
+  {
+    ROS_ERROR("Landing failed");
+    ros::shutdown();
+    return -1;
+  }
+  sleep(5);
+
+  // Takeoff
+  if (takeoff_client.call(srv_takeoff) && srv_takeoff.response.success)
+    ROS_INFO("takeoff sent %d", srv_takeoff.response.success);
```

Comment:
Make sure there's a newline after every printing. If you use `ROS_INFO()`, you don't need to worry. Capitalize the sentences that starts the line.